src/backend/executor/process.ts

```diff
@@ -106,14 +113,27 @@ export class ScheduledProcess implements Disposable {
         if (this.processParameters.forwardStdoutToLogs === undefined) {
             this.processParameters.forwardStdoutToLogs = !forwardDefaults.includes(processType);
         }
+
+        const parseLogMessage = (stdout: string) => {
+            // Do not store json output or meta messages as last error
+            const lines = stdout.split('\n')
+                .filter((line) => !line.startsWith('{') && !line.startsWith('>') && line !== '');
```

Comment:
Minor note: to play very safe, I would consider handling leading whitespaces (probably there are none, but if this changes in codecheker the parsing would still be correct).